far/platform.fs.hpp

```diff
@@ -96,6 +96,8 @@ namespace os::fs
 		DWORD Attributes{};
 		DWORD ReparseTag{};
 
+    const wchar_t *DizText{};
```

Comment:
Alright, do you have any suggestions? Here are some of the problems I ran into:
- Moving `enum_selected` to `FileList` is daunting, the method is called in far too many places on a `Panel` and it seems to be a rabbit hole without end to update all of these, also I am somewhat sceptical whether it would be possible in all cases.
- I mimiced `enum_selected` as an `enum_selected_files` method in `FileList`, but I had a lot of trouble making it work since `GetSelName` still requires an `os::fs::find_data*` as its final argument, and that's arguably not so easy to change since it's inherited.
- Even when creating another method to mimic `GetSelName`, I had troubles making this work with `make_inline_enumerator` because `FileListItem` is `NONCOPYABLE`.

In the end, the best I could come up with was an overloaded version of `GetSelName` in `FileList` which receives an additional argument `const wchar_t **strDiz` to extract the description.  In comparison, just moving `DizText` to the `find_data` seemed less ridiculous, and one can argue that a `find_result` in FAR can have a description.

Reply:
suggestion; would you prefer this?
https://github.com/huettenhain/FarManager/compare/descriptions...huettenhain:descriptions-enumerator

Reply:
Yes, that method called in quite a few places on a `Panel` and that's the problem. However, in all those places that abstract `Panel` is either an instance of a filelist (usually) or a treelist (rarely), so it's not impossible after all.
I understand your concerns though, it might require quite a few changes here and there.
I'll check if it's possible to reorganise the code and expose descriptions nicely.